File: src/sdk-reference/cpp/1/auth/update-self/snippets/update-self.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");
  kuzzleio::User updatedUser = kuzzle->auth->updateSelf(R"({"age": 42})");

  std::cout << updatedUser.content() << std::endl;
  // {"age": 42,"profileIds": ["default"]}
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.getMessage() << std::endl;
}

File: src/sdk-reference/csharp/1/auth/update-self/snippets/update-self.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  User updatedUser = kuzzle.auth.updateSelf(@"{""age"": 42}");

  Console.WriteLine(updatedUser.content());
  // {"age": 42}
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.what());
}
